compiler_opt/distributed/local/local_worker_manager.py

```diff
@@ -245,8 +248,18 @@ def __init__(self, worker_class: 'type[Worker]', count: Optional[int], *args,
                **kwargs):
     if not count:
       count = multiprocessing.get_context().cpu_count()
+    gin_config = {}
+    try:
+      gin_config = gin.get_bindings(worker_class)
+    except:  # pylint: disable=bare-except
+      pass
+    # Issue #38
+    if sys.version_info.minor >= 9:
+      final_args = kwargs | gin_config
+    else:
+      final_args = {**kwargs, **gin_config}
     self._stubs = [
-        _make_stub(worker_class, *args, **kwargs) for _ in range(count)
+        _make_stub(worker_class, *args, **final_args) for _ in range(count)
```

Comment:
Jacob also encountered this issue. One possible solution is to switch to the multiprocessing module in TFAgents that can pass gin through. but he encountered some issues after the switch. Not sure whether he solved it or now.

Reply:
This solves the problem uniformly for any distribution model, including Dask, or our internal distro framework, for example.

Reply:
(i.e. next step I'd move this in a common place)

Reply:
this would fix the issue I was running into. as for switching the multiprocessing modules, I think it's probably better to handle the issue by explicitly passing the gin flags rather than depend on subtle interactions between gin and the multiprocessing module.